uluru/data/examples/resource/initech.tps.report.v1.json

```diff
@@ -0,0 +1,78 @@
+{
+    "$id": "initech.tps.report.v1.json",
+    "typeName": "Initech::TPS::Report",
+    "description": "An example resource schema demonstrating some basic constructs and validation rules.",
+    "sourceUrl": "git@github.com:awslabs/aws-cloudformation-rpdk.git",
+    "definitions": {
+        "InitechDateFormat": {
+            "$comment": "Use the `definitions` block to provide shared resource property schemas",
+            "type": "string",
+            "format": "date-time"
```

Comment:
Is `date-time` a supported format?  When running `cfn test` it will complain about this value because it isn't in https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/resource_generator.py#L223-L259